FogCreek/Glitch-Community
# only give admins add team user button

## Links
* https://cookie-ricotta.glitch.me
* https://app.clubhouse.io/glitch/story/6822/team-members-non-admins-see-the-add-button-but-don-t-actually-have-permissions-to-add

## GIF/Screenshots:

## Changes:
* Only shows Add button when you're a team admin

## How To Test:
* Go to a team you're not an admin of, make sure there's no button to add members
* Go to a team you are an admin of, make sure the button is still there and can be used
